components/Dialogs/examples/supplemental/DialogWithPreferredContentSizeExampleViewController.m

```diff
@@ -31,10 +33,11 @@ @implementation DialogWithPreferredContentSizeExampleViewController
 - (void)viewDidLoad {
   [super viewDidLoad];
 
-  MDCButtonScheme *scheme = [[MDCButtonScheme alloc] init];
-  scheme.colorScheme = self.colorScheme;
-  scheme.typographyScheme = self.typographyScheme;
-  [MDCContainedButtonThemer applyScheme:scheme toButton:self.button];
+  if (self.containerScheme == nil) {
+    self.containerScheme = [[MDCContainerScheme alloc] init];
```

Comment:
See previous comment about consistent approach to initializing the containerScheme.

Reply:
Same as above.